prow/tide/tide.go

```diff
@@ -600,20 +595,27 @@ func accumulateBatch(presubmits map[int]sets.String, prs []PullRequest, pjs []ku
 		for _, pr := range state.prs {
 			requiredPresubmits = requiredPresubmits.Union(presubmits[int(pr.Number)])
 		}
-		passesAll := true
+		overallState := successState
 		for _, p := range requiredPresubmits.List() {
-			if s, ok := state.jobStates[p]; !ok || s != successState {
-				passesAll = false
-				log.WithField("batch", ref).Debugf("batch invalid, required presubmit %s not passing", p)
+			s, ok := state.jobStates[p]
```

Comment:
Nit: This could remain inline with the `if`

Reply:
Fixed.